org.mume.MMapper.json

```diff
@@ -0,0 +1,167 @@
+{
+  "app-id": "org.mume.MMapper",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.8",
+  "sdk": "org.kde.Sdk",
+  "finish-args": [
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--device=dri",
+    "--filesystem=xdg-documents:rw"
+  ],
+  "command": "mmapper",
+  "cleanup": [
+    "/app/images",
+    "/app/include",
+    "/app/arda.xml",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "openssl3",
```

Comment:
openssl 3.3.3 is in the runtime. why do you need this?

Reply:
I didn't realize it was in the runtime. I've removed it.